roles/contiv_network/tasks/main.yml

```diff
@@ -1,10 +1,25 @@
 ---
 # This role contains tasks for configuring and starting netmaster and netplugin service
 
+- name: remove older netmaster
+  file: path=/usr/bin/netmaster state=absent
+
+- name: remove older netplugin
+  file: path=/usr/bin/netplugin state=absent
+
+- name: remove older compose
+  file: path=/usr/bin/docker-compose state=absent
+
+- name: remove older contivctl
+  file: path=/usr/bin/contivctl state=absent
+
+- name: remove older tar file
+  file: path=/tmp/contivnet.tar.bz2 state=absent
+
 - name: download netmaster and netplugin
   get_url:
     validate_certs: no
-    url: https://github.com/contiv/netplugin/releases/download/v0.0.0-11-12-2015.00-55-50.UTC/netplugin-v0.0.0-11-12-2015.00-55-50.UTC.tar.bz2
+    url: https://cisco.box.com/shared/static/hs2gp4jfvssr1bu2geh3o6yipse9ljjd.bz2
```

Comment:
find a way to parameterize this; cluster is relying on the releases to be in there, so if you need a different url let's make it a var you can pass into ansible.